Domain Services/Health/Expense-v1.1.0.yml

```
@@ -0,0 +1,654 @@
+        fromPeriod:
+          description: Start date of bill period  
+          type: number
+          format: int64
+          example: 1680307200000
+        toPeriod:
+          description: End date of bill period  
+          type: number
+          format: int64
+          example: 1682899199000
+        netLineItemAmount:
+          description: Amount being paid for the particular bill-detail during payment  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)